tests/vmi_configuration_test.go

```diff
@@ -644,19 +645,10 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				vmi := libvmifact.NewAlpine()
 
 				By("Creating a secret with the binary ACPI SLIC table")
-				secret, err := virtClient.CoreV1().Secrets(testsuite.GetTestNamespace(vmi)).Create(
-					context.Background(),
-					&k8sv1.Secret{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:      secretWithSlicName,
-							Namespace: testsuite.GetTestNamespace(vmi),
-						},
-						Type: "Opaque",
```

Comment:
Please reason why this has been dropped.

Reply:
The secret type Opaque is the default and defined implicitly, it seemed redundant to specify it.
https://kubernetes.io/docs/concepts/configuration/secret/#opaque-secrets

Reply:
Please add it to the commit.

Reply:
Done